frontend/src/static/js/v_summary_charts.js

```diff
@@ -178,7 +178,7 @@ window.vSummaryCharts = {
         zLocation: this.getRepoLink(user),
         zSince: since,
         zUntil: until,
-        zIsMerge: isMerge,
+        zIsMerge: isMerged,
```

Comment:
shouldn't this also become sth like zIsMerged?

Reply:
Thanks for pointing it out. I have changed it accordingly.